auction-server/src/server.rs

```diff
@@ -267,6 +268,15 @@ pub async fn start_server(run_options: RunOptions) -> anyhow::Result<()> {
     };
 
     let chains_svm = setup_chain_store_svm(config_map);
+    let express_relay_svm = ExpressRelaySvm {
+        idl:                         load_express_relay_idl()?,
+        permission_account_position: env!("SUBMIT_BID_PERMISSION_ACCOUNT_POSITION")
```

Comment:
I'd rather put this inside the function that is using it. Soon, we can make these constant and fully compile time:
https://github.com/rust-lang/rust/pull/124941

Reply:
Why not here?
If the value doesn't exist, then the run will be failed.

Reply:
fair enough, but hopefully we can get rid of this struct later on when we can make these const values